drupal/web/sites/default/settings.lando.php

```diff
@@ -4,17 +4,35 @@
  * Load database credentials from Lando app environment.
  */
 $lando_info = json_decode(getenv('LANDO_INFO'), TRUE);
-$databases['default']['default'] = [
-  'driver' => 'mysql',
-  'database' => $lando_info['database']['creds']['database'],
-  'username' => $lando_info['database']['creds']['user'],
-  'password' => $lando_info['database']['creds']['password'],
-  'host' => $lando_info['database']['internal_connection']['host'],
-  'port' => $lando_info['database']['internal_connection']['port'],
-];
+
+if ($lando_info) {
+  $service_info_from_lando_env = function ($service) use ($lando_info) {
```

Comment:
What is the purpose of defining this as a function? This makes it very difficult to understand what this code does and why it is there.